package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "dbstream-mongo",
-  "version": "1.4.1",
+  "version": "1.5.0",
```

Comment:
The current release version is `v2.0.0`, don't we wanna have something like `2.1.0` here?

<img width="261" alt="Screen Shot 2022-08-31 at 6 42 26 PM" src="https://user-images.githubusercontent.com/43310906/187720976-8e9339ad-4ddc-4f00-b4de-fa246273deef.png">

```suggestion
  "version": "2.1.0",
```

Reply:
Also, it looks like the the `package-lock.json` file had been generated before you changed the version here:

<img width="475" alt="Screen Shot 2022-08-31 at 6 46 01 PM" src="https://user-images.githubusercontent.com/43310906/187722044-fe9f5f68-df4c-4422-a1f8-7f34eeef1ffb.png">